ocaml/testsuite/tests/parsing/pr6604.compilers.reference

```diff
@@ -1,4 +1,4 @@
-File "pr6604.ml", line 9, characters 0-1:
+File "pr6604.ml", line 9, characters 0-2:
 9 | #1
-    ^
-Error: Syntax error
+    ^^
+Error: The extension "layouts_alpha" is disabled and cannot be used
```

Comment:
This isn't a great error message. `#1` is not valid syntax -- we only have e.g. `#1n` and `#1L` and such. So in the case where there's no size suffix, we could probably produce a better error message.

Reply:
Oh, that's just an oversight on my part, I didn't realize that -- I can change it.  Why don't we support `#1` for `#int`?

Reply:
Oh, that's just an oversight on my part, I didn't realize that -- I can change it.  Why don't we support `#1` for `#int`?

Reply:
Because there is no `#int`: `int` is already unboxed.

Reply:
:facepalm: Of course!  And then we want people to be explicit that they want the native-sized `int`, thus `#1n`.